tools/build-benchmark.cmd

```diff
@@ -0,0 +1,35 @@
+@echo off
```

Comment:
This may no longer be needed per se. At least with latest VS2019 and `vcpkg` installed - it is no longer used.